Dockerfile.dev

```diff
@@ -19,16 +19,19 @@ RUN apt-get -q update \
 
 WORKDIR /app
 
-# The general app requirements and packages required to run Tox are installed into the system,
-# so we need to include them as part of the Docker build.
-# TODO: edit this comment
-COPY ./ /app/
-# install the requirements into the container first
-# these rarely change and is more cache friendly
-# ... really speeds up building new containers
+# The general app requirements and packages required to run Tox are installed into the system.
+# We copy them in early to avoid re-installing them unless absolutely necessary.
+COPY requirements.txt requirements-tox.txt /app/
 RUN pip install -r requirements.txt
 RUN pip install -r requirements-tox.txt
 
+COPY aus-data-snapshots/ /app/aus-data-snapshots/
+COPY auslib/ /app/auslib/
+COPY ui/ /app/ui/
+COPY scripts/ /app/scripts/
+COPY uwsgi/ /app/uwsgi/
+COPY MANIFEST.in requirements-test.txt setup.py tox.ini version.json version.txt /app/
```

Comment:
Out of curiosity, do we have to include `MANIFEST.in` and `version.json`? IIRC, the docker image isn't built out of the dev Dockerfile.